examples/add-kafkametrics-receiver/rendered_manifests/configmap-agent.yaml

```diff
@@ -95,6 +95,41 @@ data:
         - sources:
           - from: resource_attribute
             name: host.name
+      k8sattributes/metrics:
+        extract:
+          annotations:
+          - from: pod
+            key: splunk.com/sourcetype
+          - from: namespace
+            key: splunk.com/metricsIndex
+            tag_name: com.splunk.index
+          - from: pod
+            key: splunk.com/metricsIndex
+            tag_name: com.splunk.index
+          metadata:
+          - k8s.node.name
+          - k8s.namespace.name
+        pod_association:
+        - sources:
+          - from: resource_attribute
+            name: k8s.node.name
+        - sources:
+          - from: resource_attribute
+            name: k8s.pod.uid
+        - sources:
+          - from: resource_attribute
+            name: k8s.pod.ip
+        - sources:
+          - from: resource_attribute
+            name: k8s.pod.name
+        - sources:
+          - from: resource_attribute
+            name: ip
+        - sources:
+          - from: connection
+        - sources:
+          - from: resource_attribute
+            name: host.name
```

Comment:
same for this one

Reply:
I think this one does work for the case when pod's IP is set as the `host.name`

Reply:
Since we also use this association in our default k8sattribute processor too, want to confirm that it can [work with pod ip](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/k8sattributesprocessor/pod_association.go#L50)

Reply:
I'll leave it as is then

Reply:
> I think this one does work for the case when pod's IP is set as the host.name

I don't think if we still have any receiver setting pod IP to this attribute. Prometheus receiver used to set this resource attribute. It's been setting `net.host.name` for a while now which is now deprecated in sem conv https://github.com/open-telemetry/semantic-conventions/blob/bef6b4b39f8fc50c266742f56b23c7024a44bb68/docs/attributes-registry/network.md#deprecated-network-attributes. We will need to update the default config of k8sattributes processor after Prometheus receiver is updated

Reply:
Ok, so I removed host.name but kept k8s.node.name